api/placer.py

```diff
@@ -250,8 +250,8 @@ def check(self):
 
             ###
             # Remove when switch to dmv2 is complete across all gears
-            c_metadata = self.metadata.get(self.container_type, {})
-            if self.context.get('job_id') and c_metadata and not c_metadata.get('files', []):
+            c_metadata = self.metadata.get(self.container_type, {}) # pragma: no cover
+            if self.context.get('job_id') and c_metadata and not c_metadata.get('files', []): # pragma: no cover
```

Comment:
Unfortunately we are still waiting on the classifier update I believe, as it was updated to use the `classification` tag.